SYCL/ESIMD/api/functional/ctors/ctor_fill_core.cpp

```diff
@@ -0,0 +1,116 @@
+//==------- ctor_fill_core.cpp  - DPC++ ESIMD on-device test ---------------==//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+// REQUIRES: gpu, level_zero
+// XREQUIRES: gpu
+// TODO gpu and level_zero in REQUIRES due to only this platforms supported yet.
+// The current "REQUIRES" should be replaced with "gpu" only as mentioned in
+// "XREQUIRES".
+// UNSUPPORTED: cuda, hip
+// XRUN: %clangxx -fsycl %s -fsycl-device-code-split=per_kernel -o %t.out
+// XRUN: %GPU_RUN_PLACEHOLDER %t.out
+// RUN: false
+// XFAIL: *
+// TODO This test disabled due to simd<short, 32> vector filled with unexpected
+// values from 16th element. The test must be enabled when this problem will be
+// resolved.
+//
+// Test for esimd fill constructor for core types.
+
+#include "ctor_fill.hpp"
+
+using namespace sycl::ext::intel::experimental::esimd;
+using namespace esimd_test::api::functional::ctors;
+using namespace esimd_test::api::functional;
+
+int main(int argc, char **argv) {
```

Comment:
```suggestion
int main(int, char **) {
```

Reply:
I've removed it [here](https://github.com/intel/llvm-test-suite/pull/598/commits/1c807fc362443314f1e18c19bc469cffaef58e2b#diff-363883cae3ba91f2ba9d07628c42317d4da5a038cb6bef71d8f3023281b6cc79L30)